third_party/codemirror-buttons/buttons.js

```diff
@@ -1,68 +1,69 @@
 (function (mod) {
```

Comment:
We should exclude `third_party` from linting and formatting.  These are just vendored dependencies, and we're not maintaining them except to occasionally import a new upstream version.

Reply:
Yes, makes sense. I formatted and linted JS located under `web/js`. I believe there are files symlinked to the `third_party` directory?

Reply:
Yes, and that's important.  They have to be located in `third_party` for licensing purposes, but they need to be in `web` for serving.

I've found it useful to do something like `$(find ./web -type f)`, which will list only regular files and not symlinks.